src/pages/group/GroupPage.js

```diff
@@ -101,28 +102,77 @@ const GroupPage = props => {
                   <Grid.Column>
                     <Segment>
                       <Header as="h2">Topics</Header>
-                      {group.topics.map(topic => {
+                      {group.topics.map((topic) => {
                         return <Label key={topic.id}>{topic.name}</Label>;
                       })}
                     </Segment>
 
                     <Segment>
                       <Header as="h2">Resources & Links</Header>
                       <List>
-                        {group.serviceLinks.map(serviceLink => {
-                          return (
-                            <List.Item key={serviceLink.id}>
-                              <List.Icon
-                                name={LINK_TYPE_ICONS[serviceLink.type]}
-                                size="large"
-                              />
-                              <List.Content verticalAlign="middle">
-                                <a href={serviceLink.url}>{serviceLink.text}</a>
-                              </List.Content>
-                            </List.Item>
-                          );
-                        })}
+                        {group.serviceLinks
+                          .sort((a, b) =>
+                            a.isMainLink === b.isMainLink
+                              ? 0
+                              : a.isMainLink
+                              ? -1
+                              : 1,
+                          )
+                          .map((s) => {
+                            const serviceLabels = (type, url) => {
+                              const LINK_LABELS = {
+                                SLACK: 'Slack',
+                                FACEBOOK: 'Facebook',
+                                TWITTER: 'Twitter',
+                                TRELLO: 'Trello',
+                                DEVPOST: 'Devpost',
+                                VIMEO: 'Vimeo',
+                                YOUTUBE: 'Youtube',
+                                DISCORD: 'Discord',
+                                GITHUB: 'Github',
+                                INSTAGRAM: 'Instagram',
+                                TELEGRAM: 'Telegram',
+                                OTHER: 'Other',
+                                WHATSAPP: 'WhatsApp',
+                                JOGL: 'JOGL',
+                                MAIN_WEBSITE: 'Website',
+                                E_MAIL: 'E-Mail',
+                              };
+
+                              if (type === 'WEBSITE') {
```

Comment:
if you use a dictionary. It's better if you put that value on the dictionary and compare it with that.
```suggestion
                              if (type === LINK_LABELS.WEBSITE) {
```

Reply:
`LINK_LABELS` is supposed to be a dictionary for the visible labels in the app.

In your example, `LINK_LABELS.WEBSITE` would resolve to `Website`, but the API currently delivers uppercase link types – in tis cae `WEBSITE`. So I guess we have to keep it like this.